ansible-playbook-base/site.yml

```diff
@@ -85,6 +85,12 @@
   roles:
     - passwdless-ssh-mesh
 
+- name: Set gluster volume specification
```

Comment:
Move this after prepare gluster volumes, we can set options there.

We still want this as a separate step, so that we can invoke it many times for the same volume in reduced testing scenario.